README.md

```diff
@@ -144,6 +144,11 @@ This is the main Robot-Centric Elevation Mapping node. It uses the distance sens
 
         rosservice call /elevation_mapping/clear_map
 
+* **`save_map`** ([grid_map_msgs/ProcessFile])
+
+    Saves the current fused grid map and raw grid map to a rosbag file. Field `topic_name` must be a base name, i.e. no leading slash character (/). If field `topic_name` is empty, then `elevation_map` is used per default.
+
+        rosservice call /elevation_mapping/clear_map
```

Comment:
??

Reply:
the `save_map` service was not state in the readme file, even though it was implemented.

Reply:
Yeah I know, thanks for adding this. However, this will not work without adding the correct parameters require for the service request. Please verify this.